src/components/Grid/Col.jsx

```diff
@@ -79,6 +79,8 @@ Col.propTypes = {
   columnWidth: PropTypes.oneOf(validWidths),
   /** Sets the containing element. The default container element is a 'div' */
   component: PropTypes.string,
+  /** If last should the column be right aligned.*/
```

Comment:
This phrasing is odd. It reads as if it is a question. And is it about right text alignment or putting it at the end of the row/block?

Reply:
Maybe: "Place the column at the end of the row."